apps/web/pages/api/sentry-example-api/index.ts

```diff
@@ -0,0 +1,9 @@
+import { NextApiRequest, NextApiResponse } from 'next';
+
+export default function handler(req: NextApiRequest, res: NextApiResponse) {
+	try {
+		throw new Error('Sentry Example API Route Error');
```

Comment:
@NdekoCode why do we need this code / example page??

Reply:
It may happen that the sentry configuration doesn't work, so this page is generated to test whether we receive errors in the sentry dashboard. We can delete it once you've checked that the configuration on your side is working properly, sir.